src/components/Button/Button.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import './Button.scss';
+import PropTypes from 'prop-types';
+
+export default function Button(props) {
+  const {
+    className,
+    href,
+    onClick,
+    target,
+    text,
+    largerText,
+  } = props;
+
+  return href ? (
+    <a
```

Comment:
Avoid as much as possible using "a" with Gatsby : use link component instead https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/